src/app/code/community/Omise/Gateway/etc/config.xml

```diff
@@ -96,15 +96,18 @@
                 <installment>
                     <installment_bay>
                         <name>Krungsri</name>
+                        <monthly_minimum_subunits_thb>50000</monthly_minimum_subunits_thb>
                     </installment_bay>
                     <installment_first_choice>
                         <name>First Choice</name>
+                        <interest_rate>1.3</interest_rate>
                     </installment_first_choice>
                     <installment_kbank>
                         <name>KBank</name>
                     </installment_kbank>
                     <installment_bbl>
                         <name>Bangkok Bank</name>
+                        <monthly_minimum_subunits_thb>50000</monthly_minimum_subunits_thb>
```

Comment:
5k thb?

Reply:
5

Reply:
but there is zero missing on both!

Reply:
No there isn't. This is monthly minimum - 500B and 300B

Reply:
You had me worried for a second